assets/js/googlesitekit/datastore/forms/index.test.js

```diff
@@ -19,73 +19,111 @@
 /**
  * Internal dependencies
  */
-import {
-	createTestRegistry,
-	unsubscribeFromAll,
-} from 'tests/js/utils';
+import { createTestRegistry } from 'tests/js/utils';
 import { STORE_NAME } from './constants';
 
 describe( 'core/forms store', () => {
-	const formName = 'Form 1';
-	const formData = {
-		key1: 'value1',
-		key2: 'value2',
-	};
-	const newFormName = 'Form 2';
-	const newFormData = {
-		key3: 'value3',
-		key4: 'value4',
-	};
 	let registry;
 
 	beforeEach( () => {
 		registry = createTestRegistry();
 	} );
 
-	afterEach( () => {
-		unsubscribeFromAll( registry );
-	} );
-
 	describe( 'actions', () => {
 		describe( 'setValues', () => {
 			it( 'requires the formName param', () => {
 				expect( () => {
 					registry.dispatch( STORE_NAME ).setValues();
 				} ).toThrow( 'formName is required for setting values.' );
 			} );
+
 			it( 'requires the formData param', () => {
+				const formName = 'Form 1';
+
 				expect( () => {
 					registry.dispatch( STORE_NAME ).setValues( formName );
 				} ).toThrow( 'formData must be an object.' );
 			} );
+
+			it( 'requires the formData param to be an object not an array', () => {
+				const formName = 'Form 1';
+
+				expect( () => {
+					registry.dispatch( STORE_NAME ).setValues( formName, [] );
+				} ).toThrow( 'formData must be an object.' );
+			} );
+
+			it( 'requires the formData param to be an object not a string', () => {
+				const formName = 'Form 1';
+
+				expect( () => {
+					registry.dispatch( STORE_NAME ).setValues( formName, 'formData' );
+				} ).toThrow( 'formData must be an object.' );
+			} );
 		} );
 	} );
 
 	describe( 'selectors', () => {
 		describe( 'getValue', () => {
-			it( 'works with a formName not previously assigned', async () => {
-				await registry.dispatch( STORE_NAME ).setValues( formName, { ...formData } );
+			it( 'works with a formName that does not exist', () => {
+				const formName = 'Form 1';
+				const formData = {
+					key1: 'value1',
+					key2: 'value2',
+				};
+				const newFormName = 'Form 2';
+
+				registry.dispatch( STORE_NAME ).setValues( formName, { ...formData } );
```

Comment:
No need to spread the `formData` in unless you're combining objects inline. This can simply be `formData`.

Same goes for other tests.